packages/color-area/test/color-area.test.ts

```diff
@@ -131,31 +123,10 @@ describe('ColorArea', () => {
             '75%, luminosity, 67%, saturation'
         );
     });
-    it('overrides both X and Y labels with a provided "label" attribute', async () => {
```

Comment:
We should replace this with a test using `label-x` and `label-y` to customize the label.

Reply:
Done!